architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,32 +262,58 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
     - created: 2016-01-29T13:40:11Z
       dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
       generation: 1
-      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
+    tag: latest <4>
 ----
+
+<1> The name of the image stream. 
+<2> The image source repository.
+<3> The SHA identifier that this imagestream tag currently references. Resources that reference this imagestream tag use this identifier.
+<4> The image stream tag.
+
 ====
+//Ben: i'm not really sure what purpose status.dockerImageRepository even serves, since each status.tag indicates where the tag comes from. @soltysh ?
+
+
+Using image streams has several significant benefits:
+
+* You can tag, rollback a tag, and quickly deal with images, without having to re-push using the command line.
+* You can trigger Builds and Deployments when new image is pushed to the registry, or schedule the import to do it later.
+* You can share images using fine-grained access and quickly distribute images across your teams.
+* If the source image changes, the Image Stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+However, if an image author pushes a newer version of an image without properly tagging the image, it can accidentally create a new distinct image (with newer version information). 
+This condition can impact applications in unexpected ways.
+
+//Ben: elaborate on what it means to "push an image without properly tagging it" 
+
+//we need a deep discussion of reference-style tags
 
 [[image-stream-image]]
 === Image Stream Image
 
-An _image stream image_ is a virtual resource that allows you to retrieve an
-image from a particular _image stream_ where it is tagged. It is often
-abbreviated as _isimage_. It consists of two parts delimited by an at sign:
+An _image stream image_ is a virtual resource that points from within an Image Stream 
+to a particular external image ID. It is often abbreviated as _isimage_.
+
+An _image stream image_ allows you to retrieve an image from a particular image stream 
+where it is tagged.  
+
+The image stream image consists of two parts delimited by an `@` sign:
 `<image stream name>@<image name>`. To refer to the image in the
```

Comment:
`<image stream name>@<image name>` - maybe write image name/id. Although we use the name for the image object, it's actually docker image id, and people might get confused seeing name here, and seeing something like `sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d`